src/wymeditor/plugins/table/jquery.wymeditor.table.js

```diff
@@ -138,22 +138,38 @@ TableEditor.prototype.bindEvents = function () {
     jQuery(wym._box).find(
         tableEditor._options.sAddRowButtonSelector
     ).click(function () {
-        return tableEditor.addRow(wym.selectedContainer());
```

Comment:
In each of these add/remove row/column click handlers, `false` is always returned (unless there is an error in the method).

Returning `false` doesn't seem to have anything to do with the methods.
So I've plucked it out of the methods and put it unconditionally at the end of the each handler.